cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,34 @@
+# Log Level Extension
+
+
+
+## Attributes
+
+### levelname (or loglevelname)
+
+- Type: `String`
+- Description: Human readable representation of  the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be uppercase
+  - MAY be `INFO`
+  - SHOULD NOT be `INFORMATION`
+  - MAY be `ERROR`
+  - SHOULD NOT be `ERR`
+  - MAY be `WARNING`
+  - SHOULD NOT be `WARN`
+  - SHOULD have a one-to-one relationship with a distinct levelnum in the scope of the producer
+
+### levelnum (or loglevelnum)
+
+- Type: `Integer`
+
+- Description: Number representation of the level of the event, the higher the level is the event has a higher importance
```

Comment:
can you wrap this at 80 chars and remove the blank line on 28 and 32?

Reply:
Perhaps:
A numerical representation of the level of importance of the event. This specification does not define "importance" or meaning of each value other than an event with a higher numerical value MUST be of more importance than an event with a lower numerical value.